active/0014-rolling-cluster-upgrade.md

```diff
@@ -53,6 +66,13 @@ There are two major areas that need to be considered to support this kind of upg
 1. RPC compatibility
 1. Mnesia schema backward-compatibility
 
+### Design of the helper bpapi application
+
+We propose to create a new helper application that helps to maintain the API backward-compatibility.
```

Comment:
Is this application generic enough to be used in any other Erlang project ?
if it's very much tied with emqx, maybe put it in `apps/emqx/src/bpapi`

external plugins will need to add emqx (sub_dir) as a dependency anyway.

Reply:
My initial thought was that we may want to use it in Mria, for example... But perhaps it's an overkill, since we can just modify it to never try to connect to the upstream nodes running a different version.

Reply:
ok. let's do it as a part of emqx app, and split it out if it turn out to be useful as a standalone.